project/src/views/favorites/favorites.tsx

```diff
@@ -1,80 +1,40 @@
-import Logo from '../../components/logo/logo';
 import {Offer} from '../../types/offer';
-import OffersList from '../../components/offers-list/offers-list';
+import FavoritesByCity from '../../components/favorites-by-city/favorites-by-city';
 import {useState} from 'react';
+import {CITIES} from '../../const';
 
 type FavoritesProps = {
   offers: Offer[],
 }
 
-function Favorites({offers}: FavoritesProps): JSX.Element {
-  const offersFiltred = offers.filter(({isFavorite}) => isFavorite);
+export default function Favorites({offers}: FavoritesProps): JSX.Element {
   const [activeOffer, setActiveOffer] =  useState<Offer>();
+
   const hoverHandler = (offer?: Offer) => {
-    if(activeOffer !== offer){
+    if(activeOffer?.id !== offer?.id){
       setActiveOffer(offer);
     }
   };
 
+  const favoritesInCities = offers.reduce((previousValue: {[key: string]: Offer[] | []}, currentValue) => {
```

Comment:
я не понял смысла этого компонента и этой функции. ЧТо тут происходит? Зачем тебе фавориты по городам? 
```suggestion
  const favoritesInCities = offers.reduce((previousValue: {[key: string]: Offer[]}, currentValue) => {
```